azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/sync_clients.py

```diff
@@ -165,14 +188,28 @@ def __init__(self, transport):
         :param transport: The transport that the client will use.
         """
         super(DeviceClient, self).__init__(transport)
-        self._transport.on_transport_c2d_message_received = self._queue_manager.route_c2d_message
+        self._transport.on_transport_c2d_message_received = self._inbox_manager.route_c2d_message
 
-    def get_c2d_message_queue(self):
-        """Returns a MessageQueue object that can be used to receive "Cloud to Device" messages.
+    def receive_c2d_message(self, block=True, timeout=None):
```

Comment:
>block=True [](start = 34, length = 10)

making sure we're using a common/well known naming convention here? block? no_wait? something else?

Reply:
This is the same syntax as Queue

---
In reply to: [263976043](https://github.com/Azure/azure-iot-sdk-python-preview/pull/55#discussion_r263976043) [](ancestors = 263976043)